File: unit7/python/project/draw.py
from imagehelper import write_image


WIDTH = 200
HEIGHT = 200
CHANNELS = 3
MAX_COLOR_VAL = 255

PIXELS = [[[0, 0, 0] for x in range(WIDTH)] for y in range(HEIGHT)]


def main():
    # Initialize all pixels to black: (0, 0, 0)
    for y in range(0, HEIGHT):
        for x in range(0, WIDTH):
            PIXELS[y][x][0] = 0
            PIXELS[y][x][1] = 0
            PIXELS[y][x][2] = 0

    # TODO: Modify the pixels in anyway you like!

    # Save image data to file
    filename = "img/custom.ppm"
    write_image(PIXELS, filename)
    print("Wrote image to destination file: {}".format(filename))


if __name__ == "__main__":
    main()

File: unit7/java/project/Draw.java
public class Draw {

    public static final int WIDTH = 200;
    public static final int HEIGHT = 200;
    public static final int CHANNELS = 3;
    public static final int MAX_COLOR_VAL = 255;

    public static int[][][] PIXELS;

    public static void main(String[] args) {
        // Initialize all pixels to black: (0, 0, 0)
        PIXELS = new int[HEIGHT][WIDTH][CHANNELS];
        for (int y = 0; y < HEIGHT; y++) {
            for (int x = 0; x < WIDTH; x++) {
                PIXELS[y][x][0] = 0;
                PIXELS[y][x][1] = 0;
                PIXELS[y][x][2] = 0;
            }
        }

        // TODO: Modify the pixels in anyway you like!

        // Save image data to file
        String filename = "img/custom.ppm";
        ImageHelper.writeImage(PIXELS, filename);
        System.out.println("Wrote image to destination file: " + filename);
    }

}